tests/base/src/test/java/org/keycloak/test/admin/AdminConsoleLandingPageTest.java

```diff
@@ -1,33 +1,36 @@
-package org.keycloak.testsuite.admin;
+package org.keycloak.test.admin;
 
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.client.HttpClientBuilder;
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-import org.keycloak.broker.provider.util.SimpleHttp;
-import org.keycloak.representations.idm.RealmRepresentation;
-import org.keycloak.testsuite.AbstractKeycloakTest;
-import org.keycloak.testsuite.broker.util.SimpleHttpDefault;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.keycloak.test.framework.annotations.InjectKeycloakUrls;
+import org.keycloak.test.framework.annotations.KeycloakIntegrationTest;
+import org.keycloak.test.framework.server.KeycloakUrls;
+import org.keycloak.test.utils.SimpleHttpDefault;
 
 import java.io.IOException;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-public class AdminConsoleLandingPageTest extends AbstractKeycloakTest {
+@KeycloakIntegrationTest
+public class AdminConsoleLandingPageTest {
+
+    @InjectKeycloakUrls
+    KeycloakUrls keycloakUrls;
 
     private CloseableHttpClient client;
```

Comment:
We have a supplier for http-client

```suggestion
    @InjectHttpClient
    HttpClient client;
```